rules/integrations/aws/credential_access_iam_compromisedkeyquarantine_policy_attached_to_user.toml

```diff
@@ -0,0 +1,91 @@
+]
+from = "now-6m"
+index = ["filebeat-*", "logs-aws.cloudtrail-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "AWS IAM CompromisedKeyQuarantine Policy Attached to User"
+note = """
+## Triage and Analysis
+
+### Investigating AWS IAM CompromisedKeyQuarantine Policy Attached to User
+
+The AWS IAM `CompromisedKeyQuarantine` and `CompromisedKeyQuarantineV2` managed policies deny certain action and is applied by the AWS team to a user with exposed credentials. 
+This action is accompanied by a support case which specifies instructions to follow before detaching the policy. 
+
+#### Possible Investigation Steps
+
+- **Identify Potentially Compromised Identity**: Review the `userName` parameter of the `aws.cloudtrail.request_parameters` to determine the quarantined IAM entity.
+- **Contextualize with AWS Support Case**: Review any information from AWS comtaining additional information about the quarantined account and the reasoning for quarantine.
+- **Follow Support Case Instructions**: Do not revert the quarantine policy attachment or delete the compromised keys. Instead folow the instructions given in your support case.
+- **Correlate with Other Activities**: Search for related CloudTrail events before and after this change to see if the same actor or IP address engaged in potentially suspicious activities.
+- **Interview Relevant Personnel**: If the compromised key belongs to a user, verify the intent and authorization for these correlated actions with the person or team responsible for managing the compromised key.
+
+### False Positive Analysis
+
+- There shouldn't be many false positives related to this action as it is inititated by AWS in response to compromised or publicly exposed credentials.
+
+### Response and Remediation
+
+- **Immediate Review and Reversal**: Update the user IAM permissions to remove the quarantine policy and disable the compromised credentials.
+- **Policy Update**: Review and possibly update your organization’s policies on credential storage to tighten control and prevent public exposure.
+- **Incident Response**: If malicious intent is confirmed, consider it a data breach incident and initiate the incident response protocol. This includes further investigation, containment, and recovery.
+
+### Additional Information:
+
+For further guidance on managing and securing credentials in AWS environments, refer to the [AWS IAM User Guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html) regarding security best practices and guidance on [Remediating Potentially Compromised AWS Credentials](https://docs.aws.amazon.com/guardduty/latest/ug/compromised-creds.html).
+"""
+references = [
+    "https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AWSCompromisedKeyQuarantine.html/",
+    "https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AWSCompromisedKeyQuarantineV2.html/",
+]
+risk_score = 99
+rule_id = "0b79f5c0-2c31-4fea-86cd-e62644278205"
+severity = "critical"
```

Comment:
```suggestion
risk_score = 73
rule_id = "0b79f5c0-2c31-4fea-86cd-e62644278205"
severity = "high"
```

Reply:
Given that the infosec cluster contains documents related to this query, I would reconsider classifying it as critical. High/medium might make more sense, for events that can FP.